packages/chart/src/components/BarChart.tsx

```diff
@@ -352,8 +342,8 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
                       const style = applyRadius(index)
 
                       // check if bar text/value string fits into  each bars.
-                      let textWidth = getTextWidth(xAxisValue, config.fontSize)
-                      let doesTextFit = (textWidth / bar.y) * 100 < 48
+                      let textWidth = getTextWidth(xAxisValue, `normal ${fontSize[config.fontSize]}px sans-serif`)
+                      let isTextFits = textWidth < bar.y - 5 // minus padding 5
```

Comment:
nit: this could be changed to `textFits` or left as `doesTextFit`. That way when its used in an if statement its read as `if(textFits)` it makes more sense - again very small.